scripts/config/create-config.ts

```diff
@@ -0,0 +1,221 @@
+        const threshold = ethers.BigNumber.from(100000000)
+          .mul(chain.updateConditionPercentage * 100)
+          .div(10000);
+        const beaconUpdateSubscriptionConditionParameters = ethers.utils.defaultAbiCoder.encode(
+          ['uint256'],
+          [threshold]
+        );
+        const encodedBeaconUpdateSubscriptionConditions = encode([
+          {
+            type: 'bytes32',
+            name: '_conditionFunctionId',
+            value: ethers.utils.defaultAbiCoder.encode(
+              ['bytes4'],
+              [dapiServerInteface.getSighash('conditionPspBeaconUpdate')]
+            ),
+          },
+          { type: 'bytes', name: '_conditionParameters', value: beaconUpdateSubscriptionConditionParameters },
+        ]);
+        const dapiServerAddress = operationsRepository.chains[chainName].contracts.DapiServer;
+        const sponsor = chain.sponsor;
+        const beaconUpdateSubscriptionId = ethers.utils.keccak256(
+          ethers.utils.defaultAbiCoder.encode(
+            ['uint256', 'address', 'bytes32', 'bytes', 'bytes', 'address', 'address', 'address', 'bytes4'],
+            [
+              chainId,
+              airnodeAddress,
+              templateId,
+              parameters,
+              encodedBeaconUpdateSubscriptionConditions,
+              airnodeAddress,
+              sponsor,
+              dapiServerAddress,
+              dapiServerInteface.getSighash('fulfillPspBeaconUpdate'),
+            ]
+          )
+        );
+        return {
+          [beaconUpdateSubscriptionId]: {
+            chainId: `${chainId}`,
+            parameters,
+            airnodeAddress,
+            templateId,
+            conditions: encodedBeaconUpdateSubscriptionConditions,
+            relayer: airnodeAddress,
+            sponsor,
+            requester: dapiServerAddress,
+            fulfillFunctionId: dapiServerInteface.getSighash('fulfillPspBeaconUpdate'),
+          },
+        };
+      })
+    )
+    .reduce((subscriptionsObject, subscription) => ({ ...subscriptionsObject, ...subscription }), {});
+
+  const airkeeperTriggers: Triggers = {
+    rrpBeaconServerKeeperJobs: [],
+    protoPsp: Object.keys(airkeeperSubscriptions),
+  };
+
+  const airkeeperTemplates = Object.values(ConfigurationData.beacons)
+    .map((beacon: any) => beacon.template)
+    .reduce((templateObj: any, template: any) => {
+      const endpointId = ethers.utils.keccak256(
+        ethers.utils.defaultAbiCoder.encode(['string', 'string'], [template.oisTitle, template.endpointName])
+      );
+      const encodedParameters = encode(template.decodedParameters);
+      const templateId = ethers.utils.solidityKeccak256(['bytes32', 'bytes'], [endpointId, encodedParameters]);
+
+      return {
+        ...templateObj,
+        [templateId]: {
+          endpointId,
+          encodedParameters,
+        },
+      };
+    }, {}) as Templates;
```

Comment:
Now that I think about it we could add the type to the `airkeeperTemplates` instead of explicit casting, right?
```suggestion
  const airkeeperTemplates: Templates = Object.values(ConfigurationData.beacons)
    .map((beacon: any) => beacon.template)
    .reduce((templateObj: any, template: any) => {
      const endpointId = ethers.utils.keccak256(
        ethers.utils.defaultAbiCoder.encode(['string', 'string'], [template.oisTitle, template.endpointName])
      );
      const encodedParameters = encode(template.decodedParameters);
      const templateId = ethers.utils.solidityKeccak256(['bytes32', 'bytes'], [endpointId, encodedParameters]);

      return {
        ...templateObj,
        [templateId]: {
          endpointId,
          encodedParameters,
        },
      };
    }, {});
```

Reply:
Same for all other objects